src/operations/set_options.js

```diff
@@ -41,6 +41,7 @@ function weightCheckFunction(value, name) {
  * @param {string} [opts.signer.ed25519PublicKey] - The ed25519 public key of the signer.
  * @param {Buffer|string} [opts.signer.sha256Hash] - sha256 hash (Buffer or hex string) of preimage that will unlock funds. Preimage should be used as signature of future transaction.
  * @param {Buffer|string} [opts.signer.preAuthTx] - Hash (Buffer or hex string) of transaction that will unlock funds.
+ * @param {string} [opts.signer.ed25519SignedPayload] - Signed payload signer (ed25519 public key + raw payload) for atomic transaction signature disclosure.
```

Comment:
Maybe we need to add corresponding changes in `index.d.ts`.

Reply:
Good point, thanks!
Updated typescript bindings in f4651697bb91bfbfe33add452bda50adaad1a0bb